scripts/badger/process_bribes.py

```diff
@@ -0,0 +1,118 @@
+)
+CVX = interface.ERC20(registry.eth.treasury_tokens.CVX, owner=SAFE.account)
+
+# percentage of the bribes that is used to buyback $badger
+BADGER_SHARE = .275
+# percentage of the bribes that are dedicated to the treasury
+OPS_FEE = .05
+
+
+def step0_1(sim=False):
+    strat_logic = interface.IVestedCvx(
+        registry.eth.logic['native.vestedCVX'], owner=SAFE.account
+    )
+    bribes_dest = GreatApeSafe(strat_logic.BRIBES_PROCESSOR())
+    bribes_dest.take_snapshot(registry.eth.bribe_tokens_claimable.values())
+
+    if sim:
+        from brownie_tokens import MintableForkToken
+        alcx = MintableForkToken(registry.eth.treasury_tokens.ALCX)
+        alcx._mint_for_testing(SAFE.badger.strat_bvecvx, 500e18)
+        claimed = {alcx.address: 500e18}
+    else:
+        claimed = SAFE.badger.claim_bribes_votium(
+            registry.eth.bribe_tokens_claimable
+        )
+    for addr, mantissa in claimed:
+        order_payload, order_uid = SAFE.badger.get_order_for_processor(
+            sell_token=SAFE.contract(addr),
+            mantissa_sell=mantissa,
+            buy_token=WETH,
+            coef=.99,
+            prod=COW_PROD
+        )
+        PROCESSOR.sellBribeForWeth(order_payload, order_uid)
+        SAFE.badger.claim_bribes_convex(registry.eth.bribe_tokens_claimable)
+
+    bribes_dest.print_snapshot()
+
+    SAFE.post_safe_tx(call_trace=True)
+
+
+def step0(sim=False):
+    # can be skipped if step0_1 was successful
+    strat_logic = interface.IVestedCvx(
+        registry.eth.logic['native.vestedCVX'], owner=SAFE.account
+    )
+    bribes_dest = GreatApeSafe(strat_logic.BRIBES_PROCESSOR())
+    bribes_dest.take_snapshot(registry.eth.bribe_tokens_claimable.values())
+
+    if sim:
+        from brownie_tokens import MintableForkToken
+        alcx = MintableForkToken(registry.eth.treasury_tokens.ALCX)
+        alcx._mint_for_testing(SAFE.badger.strat_bvecvx, 500e18)
+    else:
+        SAFE.badger.claim_bribes_votium(registry.eth.bribe_tokens_claimable)
+        SAFE.badger.claim_bribes_convex(registry.eth.bribe_tokens_claimable)
+
+    bribes_dest.print_snapshot()
+
+    SAFE.post_safe_tx(call_trace=True)
+
+
+def step1():
+    # can be skipped if step0_1 was successful
+    want_to_sell = registry.eth.bribe_tokens_claimable.copy()
+    want_to_sell.pop('CVX') # SameBuyAndSellToken
+    want_to_sell.pop('MTA') # dust
+    want_to_sell.pop('NSBT') # dust
+    for _, addr in want_to_sell.items():
+        token = SAFE.contract(addr)
+        balance = int(token.balanceOf(SAFE.badger.strat_bvecvx))
+        if balance == 0:
+            continue
+        order_payload, order_uid = SAFE.badger.get_order_for_processor(
+            sell_token=token,
+            mantissa_sell=balance,
+            buy_token=WETH,
+            coef=.99,
+            prod=COW_PROD
+        )
+        PROCESSOR.sellBribeForWeth(order_payload, order_uid)
+
+
+def step2():
+    badger_share = int(WETH.balanceOf(SAFE) * BADGER_SHARE)
```

Comment:
here i think also requires to check instead the `PROCESSOR` balance after `sellBribeForWeth` is exec